### PR TITLE
style: makes unused vars error with linting

### PR DESCRIPTION
**Pull request by jackcmeyer**
**Changes proposed in this pull request:**

- makes unused variables error with linting

### CODE REVIEW

File: src/clients/db/Repository.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* eslint "@typescript-eslint/camelcase": "off" */
+/* eslint "@typescript-eslint/camelcase": "off", "@typescript-eslint/no-unused-vars": "off" */
```
Comment (jackcmeyer) [inline src/clients/db/Repository.ts]: In general we want to not have unused variables in our code, however, in this file we are restructuring generic objects and ensuring we don't copy over some ids, which is why I have ignored it in this file and this file only.

**Comment by tehKapa** [comment]
@Jack, I found this reference https://stackoverflow.com/questions/57802057/eslint-configuring-no-unused-vars-for-typescript 
we're missing an extension https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs

File: src/clients/db/Repository.ts (hunk 1 above)
Comment (fox1t) [reply src/clients/db/Repository.ts]: Add this rule https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings to eslintrc instead.
` { "ignoreRestSiblings": true }`
On object destructuring is always safe to assume that we just want to get rid of some props.

File: .eslintrc.js [hunk 2]
```diff
@@ -7,7 +7,9 @@ module.exports = {
   },
   extends: [
     'airbnb',
-    'plugin:@typescript-eslint/recommended',
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
```
Comment (fox1t) [inline .eslintrc.js]: What's the difference between typescript-eslint/eslint-recommended and typescript-eslint/recommended? I really can't get it.

**Pull request merged**